packages/react-components/lib/robots/robot-info.tsx

```diff
@@ -15,6 +15,16 @@ function getTaskStatusDisplay(assignedTask?: string, taskStatus?: string) {
   }
 }
 
+function calculateLastUpdateTime(updateTime: number): string {
```

Comment:
I propose having more lenient ranges like `< 5s ago`, `< 30s ago`, `> 1m ago`, `> 5m ago`, this will hopefully also reduce the amount of re-rendering needed.

Reply:
I found out why you had the issue, its a miss type from my side, thanks.
about the ranges, it is a good idea, so the update interval wouldn't be a constant one second, but a number that increases exponentially with time?

Reply:
Per VC, this will be resolved when we look into using `data-fns` but for now a lower update rate has made it more consistent